Dockerfile

```diff
@@ -28,6 +35,7 @@ RUN apt-get update -y && apt-get install -y \
     make \
     && ln -s /opt/bats/bin/bats /usr/local/bin/bats \
     && make -C /${PROJECT_NAME} install \
```

Comment:
Now that all the tools are being installed through the Dockerfile, this line calling the `install` target should be removed. That will fix the travis-ci failure.